### DISCUSSION TITLE
建议：子目录树展示（像Windows或者永硕E盘）

### DISCUSSION THREAD

**Discussion by ctem049** [spencerwooo/onedrive-vercel-index#191] [Ideas]
子目录显示树的形式比较方便

Windows资源浏览器左边目录树
永硕E盘例子：http://ygocore.ys168.com/

**Comment by myl7** [comment]
对于 OneDrive 而言比较困难……OneDrive API 没有办法高效地获取一个树状的目录结构，只能一层一层地访问
